nuxeo-features/nuxeo-platform-userworkspace/nuxeo-platform-userworkspace-core/src/test/java/org/nuxeo/ecm/platform/userworkspace/core/tests/TestUserWorkspace.java

```diff
@@ -359,12 +364,14 @@ public void testCanRetrieveUserWorkspaceWithTrashedDomain() {
                                         .collect(Collectors.toList());
         TrashService trashService = Framework.getService(TrashService.class);
         trashService.trashDocuments(docs);
+        bulkService.await(Duration.ofSeconds(30));
```

Comment:
You ask `CoreFeature#waitForAsyncCompletion` instead to avoid having `BulkService` here.